javascript/game.js

```diff
@@ -1,117 +1,69 @@
+$(document).ready(function () {
```

Comment:
Until we start using build tools, you'll always want to wrap your code in either a 

`$(document).ready(function(){ // code goes here })`

or an IIFE (Immediately Invoked Function Expression)

`(function(){ // code goes here })()`

Otherwise all of your javascript variables will be available for somebody to manipulate in the console (setting wins to 1,000,000 for instance 🙃).